src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        int arraySize = 6; // Можна змінити на інший розмір
```

Comment:
All magic numbers in your code should be constants. Consider defining '6' as a constant to give it a meaningful name.